### DISCUSSION TITLE
Any way to access the next event on the calendar?

### DISCUSSION THREAD

**Discussion by Roky3029** [theOehrly/Fast-F1#497] [Q&A]
Is there a way to get the next event on the calendar? I mean, get the next GP, or testing session automatically. 
For example: the next event on the calendar would be the Pre-season testing session, or, if we only count GP, the Bahrain GP. I'd like to know if there's a way to get this type of data.
If I haven't made myself clear, ask for more details, please.

**Comment by theOehrly** [comment]
In general, you can use https://docs.fastf1.dev/fastf1.html#fastf1.get_events_remaining to get the schedule for the remaining season. Then you just take the first event in that dataframe and you will have the next event.

This function assumes the current year for the season, therefore, at the moment you would still manually need to provide a date like 01/01/2024 so that the 2024 season is used.
